session-10/task/channels_basic_operations/task4.go

```diff
@@ -0,0 +1,20 @@
+package channels_basic_operations
+
+import "fmt"
+
+func sendToChannel1(val []int, ch chan []int) {
+	ch <- val
+}
+
+func RunTask4() {
+	ch := make(chan []int)
+	defer func() {
+		close(ch)
+		fmt.Println("Channel closed")
+	}()
+	go sendToChannel1([]int{1, 2, 3, 4, 5}, ch)
+	numbers := <-ch
+	for number := range numbers {
+		fmt.Println(number + 1)
+	}
```

Comment:
Burda numbers uzerinden range vererken key ve value olaraq deyeri goture bilerik. Tek deyer key (index)-i goturduyune gore cox guman +1 ile gedirsen
```suggestion
        for _, v := range numbers {
		fmt.Println(v)
	}
```